src/main.cc

```diff
@@ -77,6 +77,21 @@ int wmain(int argc, const wchar_t* argv[]) {
 
       if (!updater.SetIcon(argv[++i]))
         return print_error("Unable to set icon");
+    } else if (wcscmp(argv[i], L"--set-requestedExecutionLevel") == 0 || 
```

Comment:
Should this be `--set-requested-execution-level`? That is what is mentioned in the `README.md`.